features/ember-jquery.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable no-console */
```

Comment:
I think I would have expected the optional feature to be named `jquery-integration`, thoughts?

Mentioned in https://github.com/emberjs/ember-optional-features/issues/3 and https://github.com/ember-cli/ember-cli/issues/7842.